testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -130,6 +160,82 @@ class TestOpenMMTopologyParser(OpenMMTopologyBase):
     expected_n_bonds = 1922
 
 
+class TestOpenMMTopologyParserWithNoElements(OpenMMTopologyBase):
+    ref_filename = app.PDBFile(PDB).topology
+    for a in ref_filename.atoms():
+        a.element = None
+
+    expected_n_atoms = 47681
+    expected_n_residues = 11302
+    expected_n_segments = 1
+    expected_n_bonds = 25533
+
+    expected_attrs = [
+        "ids",
+        "names",
+        "resids",
+        "resnames",
+        "masses",
+        "bonds",
+        "chainIDs",
+    ]
+
+    def test_no_elements_warn(self, top):
+        wmsg = ("Element information is missing, elements attribute "
+                "will not be populated. "
+                "Atomtype attribute will be guessed using atom "
+                "name and mass will be guessed using atomtype."
+                "See MDAnalysis.topology.guessers.")
+
+        with pytest.warns(UserWarning, match=wmsg):
+            mda_top = self.parser(self.ref_filename).parse()
+
+
+class TestOpenMMTopologyParserWithPartialElements(OpenMMTopologyBase):
```

Comment:
Does this class need to subclass `OpenMMTopologyBase`? Do you use any of the base class tests or fixtures?

Reply:
Now that you pointed it out, it doesn't need to subclass. Got rid of many redundant tests after removing it.